run_tensorboard.sh

```diff
@@ -11,7 +13,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 gsutil cp $GCS_PROXY_CONFIG /etc/oauth2_proxy.cfg
 export OAUTH2_PROXY_COOKIE_SECRET=`python -c 'import os,base64; print base64.b64encode(os.urandom(16))'`
 oauth2_proxy -http-address="0.0.0.0:8080" -upstream="http://127.0.0.1:8081" -config=/etc/oauth2_proxy.cfg 2>&1 &
-tensorboard --logdir $EVENT_FILE_PATH --reload_interval $RELOAD_INTERVAL --debug --port 8081 --host 127.0.0.1
+tensorboard --logdir $EVENT_FILE_PATH --reload_interval $RELOAD_INTERVAL --port 8081 --host 127.0.0.1
```

Comment:
Lets actually change this to forward all arguments to the script to the tensorboard command, that way users can decide whether they want, e.g. debug level logging or not

Reply:
You mean adding port, host and debug as ENV variables in `app.yaml`? btw, I've removed "--debug" as there is not such command with the current version and was crashing the startup script. The closest are `--debugger_data_server_grpc_port` and `--debugger_port` but I believe it's something different

Reply:
Ahh I thought you were removing to reduce verbosity. Weird that there isn't a way top control log level...